proxy/IPAllow.cc

```diff
@@ -195,24 +253,203 @@ IpAllow::Print()
 int
 IpAllow::BuildTable()
 {
-  int file_size = 0;
-  int line_num  = 0;
-  IpAddr addr1;
-  IpAddr addr2;
-  bool alarmAlready = false;
-  ts::LocalBufferWriter<1024> bw_err;
-
   // Table should be empty
   ink_assert(_src_map.count() == 0 && _dst_map.count() == 0);
 
-  file_buff = readIntoBuffer(config_file_path, "ip-allow", &file_size);
+  std::error_code ec;
+  ts::file::path fpath{config_file_path};
```

Comment:
Change `config_file_path` from `ats_scoped_str` to `ts::file::path` and skip this step?